single_tech_samples/synapseanalytics/sample3_integration_testing/infrastructure/modules/log_analytics.bicep

```diff
@@ -0,0 +1,32 @@
+param project string
+@allowed([
+  'dev'
+  'stg'
+  'prod'
+])
+param env string
+param location string = resourceGroup().location
+param deployment_id string
+param retentionInDays int = 31
+
+
+resource loganalyticsworkspace 'Microsoft.OperationalInsights/workspaces@2020-08-01' = {
+  name: '${project}-log-${env}-${deployment_id}'
+  location: location
+  tags: {
+    DisplayName: 'Log Analytics'
+    Environment: env
+  }
+  properties: {
+    sku: {
+      name: 'PerGB2018'
+    }
```

Comment:
I haven't seen cost differences, but please take a look to see if this is the least expensive SKU for this sample to function as intended.

![image](https://user-images.githubusercontent.com/9535962/207622421-6d6ca92f-a2a8-4a89-97b7-780fe5973460.png)

Reply:
I'm removing the Log Analytics resources from this sample as it's not necessary for this section.